tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryResultValueComparator.java

```diff
@@ -186,22 +191,19 @@ private int longEqual(Object actual, Object expected)
         return actualLong.compareTo(expectedLong);
     }
 
-    private static int floatingEqualWithTolerance(Object actual, Object expected, double tolerance)
+    private int floatingEqual(Object actual, Object expected)
     {
         if (!(isFloatingPointValue(actual) && isFloatingPointValue(expected))) {
             return -1;
         }
-        return DoubleMath.fuzzyCompare(getDoubleValue(actual), getDoubleValue(expected), tolerance);
-    }
-
-    private int singleFloatingEqual(Object actual, Object expected)
-    {
-        return floatingEqualWithTolerance(actual, expected, MACHINE_EPSILON_FLOATING_32B);
-    }
 
-    private int doubleEqual(Object actual, Object expected)
-    {
-        return floatingEqualWithTolerance(actual, expected, MACHINE_EPSILON_FLOATING_64B);
+        double expectedDouble = getDoubleValue(expected);
+        double tolerance = 0;
```

Comment:
I believe machine epsilon is a better default since tolerance = 0 does not have sense in double comparison by definition.